R/compute_weighted_intensity_levels.R

```diff
@@ -66,11 +55,11 @@
 #' )
 #'
 #' # Use the peak model
-#' fit_peak(weighted_observations = peak_input,
-#'          conf_levels = c(0.50, 0.90, 0.95),
-#'          family = "weibull")
+#' compute_weighted_intensity_levels(weighted_observations = peak_input,
```

Comment:
I think you should not have a default value for `weighted_observations`.  Where is the `peak_input` object defined?

Reply:
Is it not fine to have in the example?

It is not as default in the function:
compute_weighted_intensity_levels <- function(
  weighted_observations,
  conf_levels = c(0.50, 0.90, 0.95),
  family = c("weibull",
             "lnorm",
             "exp"),
  optim_method = c("Nelder-Mead",
                   "BFGS",
                   "CG",
                   "L-BFGS-B",
                   "SANN",
                   "Brent"),
  lower_optim = -Inf,
  upper_optim = Inf
)